src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -622,6 +622,23 @@ def listDataRulesHistory(self, name, scope='cms'):
             self.logger.error("Exception listing rules history for data: %s. Error: %s", name, str(ex))
         return list(res)
 
+    def listParentDIDs(self, name, scope='cms'):
+        """
+        _listParentDID__
+
+        List the parent block/container of the specified DID.
+        :param name: data identifier (either a block or a file name)
+        :param scope: string with the scope name
+        :return: a list with dictionary items
+        """
+        res = []
+        try:
+            res = self.cli.list_parent_dids(scope, name)
+        except Exception as ex:
+            self.logger.error("Exception listing parent DIDs for data: %s. Error: %s", name, str(ex))
+        return list(res)
```

Comment:
Are you casting it to a list because it returns a generator?

Reply:
Yes, it does return a generator